vllm/multimodal/image.py

```diff
@@ -0,0 +1,120 @@
+from typing import Dict, Type
+
+import numpy as np
+import torch
+from PIL import Image
+
+from vllm.config import ModelConfig, VisionLanguageConfig
+from vllm.logger import init_logger
+from vllm.sequence import SequenceData
+from vllm.transformers_utils.image_processor import cached_get_image_processor
+
+from .base import MultiModalData, MultiModalPlugin
+
+logger = init_logger(__name__)
+
+
+def _get_seq_data(seq_len: int, vlm_config: VisionLanguageConfig):
+    token_ids = [vlm_config.image_token_id] * vlm_config.image_feature_size
+    token_ids += [0] * (seq_len - vlm_config.image_feature_size)
+
+    return SequenceData(token_ids)
```

Comment:
Let's add a `NOTE` that this currently assumes image data is simply concatenated with prompt.